### PR TITLE
Retrive user cvs#95

### PR DESCRIPTION
**Pull request by CharlesKimMaina**
#Endpoint to retrieve the last uploaded CVs

Please provide a short summary explaining what this PR is about.

Fixes https://github.com/HackYourFuture-CPH/rate-my-cv/issues/95

# How to test?

Please provide a short summary how your changes can be tested?
npm run dev and add limit as query parameter like in below
http://localhost:3000/api/usercv?limit=2

limit_query_added

limitn_query_on_localhost

# Checklist

- [*] I have performed a self-review of my own code
- [*] I have commented my code, particularly in hard-to-understand areas
- [*] I have made corresponding changes to the documentation
- [*] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/api/controllers/users.controller.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
-/* TODO: This is an example controller to illustrate a server side controller.
-Can be deleted as soon as the first real controller is added. */
+/*  This is the controller for cvs */
 
 const knex = require('../../config/db');
+
 const Error = require('../lib/utils/http-error');
 
 const moment = require('moment-timezone');
+const { request } = require('express');
```
Comment (bytewiz) [inline src/server/api/controllers/users.controller.js:8]: This remove be removed :) as it is not used

File: src/server/api/routes/api-router.js [hunk 2]
```diff
@@ -41,6 +41,6 @@ router.use('/cv', cvsRouter);
 
 router.use('/users', usersRouter);
 
-router.use('/usercv', usercvRouter);
+router.use('/usercv', usercvRouter); //newest route
```
Comment (bytewiz) [inline src/server/api/routes/api-router.js:44]: don't think we need that comment :)

**Review by bytewiz — CHANGES_REQUESTED** [review]
Looks good 👍  just some quick things to clean up then I will approve 👍

File: src/server/api/controllers/users.controller.js (hunk 1 above)
Comment (CharlesKimMaina) [reply src/server/api/controllers/users.controller.js:8]: yap true, let me remove it

**Comment by CharlesKimMaina** [comment]
I will create a new PR instea

**Comment by CharlesKimMaina** [comment]
I am creating another PR as this has extra files i din't commit which sneaked in

**Review by bytewiz — APPROVED** [review]
Seems fine to me 👍

**Pull request merged**